Makefile

```diff
@@ -231,9 +236,15 @@ copyright:
 	@printf $(COLOR) "Fix license header..."
 	@go run ./cmd/tools/copyright/licensegen.go
 
-lint:
+goimports: update-goimports
+	@printf $(COLOR) "Run goimports for modified files..."
+	@printf "Merge base: $(MERGE_BASE)\n"
+	@printf "Modified files: $(MODIFIED_FILES)\n"
+	@goimports -w $(filter %.go, $(MODIFIED_FILES))
+
+lint: update-linters goimports
 	@printf $(COLOR) "Run linters..."
-	@golangci-lint run --verbose --timeout 10m --fix=false --new-from-rev=HEAD~ --config=.golangci.yml
+	@golangci-lint run --verbose --timeout 10m --fix=false --new-from-rev=$(MERGE_BASE) --config=.golangci.yml
```

Comment:
Could you try just changing this to `--fix=true`? That should fix the problem you're pointing out where we don't have a command to fix `goimports` errors.

Reply:
Good catch!
That is probably a better alternative!